packages/ui/src/components/form-builder/Forms/CreateBusiness/CreateBusinessForm.tsx

```
@@ -0,0 +1,108 @@
+	mutate: ReturnType<typeof useCreateBusiness>["mutate"];
+	isPending: boolean;
+	error: Error | null;
+	isOpen: boolean;
+};
+
+// remove unused keys
+export const businessFormConfig: ConfigObject<CreateBusinessInput> = {
+	name: {
+		name: "name",
+		input: "text",
+		label: texts["EN"].businessNameLabel,
```

Biome (1.9.4)

[error] The computed expression can be simplified without the use of a string literal.

Unsafe fix: Use a literal key instead.


(lint/complexity/useLiteralKeys)